### DISCUSSION TITLE
Is there a way to make integrations for all forms?

### DISCUSSION THREAD

**Discussion by johncadengo** [heyform/heyform#55] [Ideas]
Right now, it appears that you can "Connect" integrations per form. So every time you make a new form, you can go into connect, and then pick an integration. Can we make a project-wide or workspace-wide integration? In other words, can I make an integration that applies to all forms within a project or workspace?

**Comment by iMuFeng** [comment]
Only email notification and webhook can be applied to all forms. For example, with platforms like Airtable, where form fields need to correspond to table fields, each form must be set up individually.

**Reply by johncadengo** [reply]
Okay, that makes sense. Is there an option to do that or are you saying it'd have to be a new feature?

**Reply by iMuFeng** [reply]
This can be added as a new feature.
